winml/test/scenario/cppwinrt/scenariotestscppwinrt.cpp

```diff
@@ -136,7 +136,10 @@ static void Scenario1LoadBindEvalDefault() {
   std::wstring filePath = FileHelpers::GetModulePath() + L"model.onnx";
   LearningModel model = LearningModel::LoadFromFilePath(filePath);
   // create a session on the default device
-  LearningModelSession session(model, LearningModelDevice(LearningModelDeviceKind::Default));
+  auto device = LearningModelDevice(LearningModelDeviceKind::Default);
+  LearningModelSession session(model, device);
+  LearningModelSession session2(model, device);
+  LearningModelSession session3(model, device);
```

Comment:
Is this just to show that you can create multiple sessions with the same device, or are sessions 2 and 3 used for something else?

Reply:
just to create the sessions - and the thread pools